package.json

```diff
@@ -46,14 +47,14 @@
     "express": "^4.14.0",
     "inflection": "^1.6.0",
     "loopback-connector-remote": "^1.0.3",
-    "loopback-datasource-juggler": "^3.0.0-alpha.1",
+    "loopback-datasource-juggler": "^3.0.0-alpha.7",
```

Comment:
This is clashing with https://github.com/strongloop/loopback/pull/2680 Let's land my PR first and then rebase your on top of the new master.